content/blog/2023-09-01-almalinux-events-2023.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "AlmaLinux Live: Coming to a city near you"
+type: blog
+author: 
+ name: "AlmaLinux Project"
+ bio: "-"
+ image: /users/AlmaLinux.png
+date: 2023-09-02
+post: 
+    title: "AlmaLinux Live: Coming to a city near you"
+    image: /blog-images/2022_ATO_Astro_1200x628.jpg
+---
+
+Want to learn more about the AlmaLinux OS and AlmaLinux OS Foundation? Good news! We're going to be attending a number of events through the remainder of 2023, and we might just be in your backyard. 
+
+AlmaLinux will have a presence at a number of popular events coming up soon, including:
+
+ * [Ohio LinuxFest (OLF)](http://olfconference.org/) in Columbus Ohio: September 8th and 9th.
+ * [All Things Open](http://allthingsopen.org/) in Raleigh, North Carolina: October 15th through October 18th.
+ * [LinuxFest Northwest](https://linuxfestnorthwest.org/) in Bellingham, Washington: October 20th through October 22nd. 
+ * [SeaGL](https://seagl.org/) in Seattle, Washington: November 3rd and November 4th.
+ * [SC (formerly SuperComputing)](https://supercomputing.org/) in Denver, Colorado: November 12th through November 17th. 
+
+## Talks at OLF and ATO
+
+You'll also be able to catch talks by AlmaLinux OS Foundation Chair benny Vasquez at OLF and All Things Open. At OLF benny will be speaking on Saturday, September 9th at 10:00 a.m. on "The Future of AlmaLinux: Building without Following." 
```

Comment:
```suggestion
You'll also be able to catch talks by AlmaLinux OS Foundation Chair benny Vasquez at OLF and All Things Open. 

At OLF benny will be speaking on Saturday, September 9th at 10:00 a.m. on "The Future of AlmaLinux: Building without Following." 
> Since our inception, AlmaLinux OS has been an exact duplicate of Red Hat Enterprise Linux. With RedHat's shift away from sharing the full building blocks on [git.centos.org](http://git.centos.org/), the AlmaLinux team has been hard at work to define a new, reliable path forward. Come hear about what we've been doing since June, and what we're planning for the future.


```